picker_wrapper.go

```diff
@@ -60,6 +70,19 @@ func (bp *pickerWrapper) connectionError() error {
 	return err
 }
 
+func (bp *pickerWrapper) updateStickinessMDKey(newKey string) {
+	bp.stickinessMu.Lock()
+	if bp.stickinessMDKey != newKey {
```

Comment:
Can `stickinessMDKey` be updated on the fly? Currently, it seems like this method is called only during initialization.